### PR TITLE
Add Lightbulb

### PR DESCRIPTION
**Pull request by Tyrrrz**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/1431)

### CODE REVIEW

File: manifests/Tyrrrz/LightBulb/2.1.0.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Id: tyrrrz.lightbulb
```
Comment (MarNicGit) [inline manifests/Tyrrrz/LightBulb/2.1.0.yaml:1]:
```suggestion
 Id: Tyrrrz.LightBulb
```

Package names are generally in [PascalCase](https://techterms.com/definition/pascalcase).

File: manifests/Tyrrrz/LightBulb/2.1.0.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Id: Tyrrrz.LightBulb
+Publisher: Tyrrrz
+Name: LightBulb
+AppMoniker: lightbulb
+Author: Alexey Golub
+Version: 2.1.0
+Homepage: https://github.com/Tyrrrz/LightBulb
+License: Copyright (C) 2017-2020 Alexey Golub
```
Comment (samiranshah) [inline manifests/Tyrrrz/LightBulb/2.1.0.yaml:8]: please append "GNU General Public License" too

Comment (Tyrrrz) [reply manifests/Tyrrrz/LightBulb/2.1.0.yaml]: Updated

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**